unified-runtime/source/adapters/opencl/context.hpp

```diff
@@ -42,6 +46,13 @@ struct ur_context_handle_t_ {
                                     const ur_device_handle_t *phDevices,
                                     ur_context_handle_t &Context);
   ~ur_context_handle_t_() {
+    // If we're reasonably sure this context is about to be destroyed we should
+    // clear the ext function pointer cache. This isn't foolproof sadly but it
+    // should drastically reduce the chances of the pathological case described
+    // in the comments in common.hpp.
```

Comment:
the pathological case is no longer described in the comments in common.hpp

Reply:
I assumed it referred to this:
https://github.com/intel/llvm/blob/sycl/unified-runtime/source/adapters/opencl/common.hpp#L338

Reply:
ah yeah thought the deleted comment was the only one